server/plugin/plugin.go

```diff
@@ -250,6 +263,86 @@ func (p *Plugin) OnDeactivate() error {
 	return nil
 }
 
+func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	githubEmoji := p.emojiMap[reaction.EmojiName]
+	if githubEmoji == "" {
+		return
+	}
+	post, error := p.client.Post.GetPost(reaction.PostId)
```

Comment:
`error` is a an existing "type" in Go, so we shouldn't shadow it here

```suggestion
	post, err := p.client.Post.GetPost(reaction.PostId)
```